### PR TITLE
*: Cut v0.16.2

### PR DESCRIPTION
**Pull request by mxinden**

### CODE REVIEW

**Comment by stuartnelson3** [comment]
seems to be erroring regularly, but the output message isn't super helpful.. something with errcheck?

**Comment by simonpasquier** [comment]
We would need #1825 too.

**Comment by simonpasquier** [comment]
IIUC this would need https://github.com/prometheus/alertmanager/pull/1747 too.

**Comment by mxinden** [comment]
I won't be able to do the release today. If someone wants to go ahead, feel free. Otherwise I will try to do it tomorrow.

**Comment by mxinden** [comment]
This pull request includes https://github.com/prometheus/alertmanager/pull/1822, https://github.com/prometheus/alertmanager/pull/1825 and to fix CI https://github.com/prometheus/alertmanager/pull/1747.

While cherry-picking https://github.com/prometheus/alertmanager/pull/1825 I did not include `TestPagerDutyErr` and `TestTruncate` in https://github.com/prometheus/alertmanager/blob/master/notify/impl_test.go, as it is not part of the original change set of any of the above mentioned patches. @simonpasquier let me know if you think differently.

@simonpasquier @stuartnelson3 @brian-brazil this is ready for another review if you have time.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.16.2 / 2019-04-03
+
+Updating to v0.16.2 is recommended for all users using the Slack notifier, as
```
Comment (mxinden) [inline CHANGELOG.md]: Adjusting that in a minute to the other receivers as well.

**Comment by simonpasquier** [comment]
Maybe you could cherry-pick #1739 as it seems to hit quite a few people (https://github.com/prometheus/alertmanager/pull/1739#issuecomment-480202938)?

**Comment by mxinden** [comment]
Is https://github.com/prometheus/alertmanager/pull/1739 a bug fix? If not I would suggest including it in `v0.17.0`. I am happy to kick off a pull request for that next. Would that be an option as well @simonpasquier?

**Comment by isantospardo** [comment]
yes please, go ahead, I would be really happy to get #1739 in 0.16.2

**Comment by mxinden** [comment]
I have included https://github.com/prometheus/alertmanager/pull/1739. Given that @metazool seemed to have changed their e-mail address in Github, DCO fails. I suggest we ignore the failure given that the DCO process was successful in https://github.com/prometheus/alertmanager/pull/1739.

DCO bot error:

> Commit sha: 03f7459, Author: Jo Walsh, Committer: Max Leonard Inden; Can not find "Jo Walsh 369631+metazool@users.noreply.github.com", in ["Jo Walsh jowalsh@bgs.ac.uk", "Jo Walsh jowalsh@bgs.ac.uk", "Jo Walsh jowalsh@bgs.ac.uk", "Jo Walsh jowalsh@bgs.ac.uk", "Simon Pasquier spasquie@redhat.com"].

**Comment by simonpasquier** [comment]
> I suggest we ignore the failure given that the DCO process was successful in #1739.

:+1: it happens often for release branches.

**Comment by mxinden** [comment]
Alright, ignoring the DCO issue then.

Any other comments @simonpasquier?

**Comment by simonpasquier** [comment]
LGTM

**Pull request merged**